backend/tests/validation/setup_google_calendar_test_data.py

```
@@ -0,0 +1,262 @@
+                    "overrides": [
+                        {"method": "popup", "minutes": 15},
+                    ],
+                },
+            },
+            # Event 4: Simple event without attendees (for basic testing)
+            {
+                "summary": "Parity Test Simple Event",
+                "description": "A simple event for testing",
+                "location": "Office",
+                "start": {
+                    "dateTime": (tomorrow.replace(hour=14, minute=0)).isoformat(),
```

Ruff (0.14.13)

Undefined name `tomorrow`

(F821)